src/garage/torch/_functions.py

```diff
@@ -383,6 +383,61 @@ def state_dict_to(state_dict, device):
     return state_dict
 
 
+# pylint: disable=abstract-method
+class Module(nn.Module):
+    """Wrapper class for Garage PyTorch modules."""
+
+    def __getstate__(self):
+        """Save the current device of the module before saving module state.
+
+        Returns:
+            dict: State dictionary.
+        """
+        # do we always run experiments on global device?
+        save_from_device = global_device()
+        self.to('cpu')
+        state = self.__dict__.copy()
+        state['device'] = save_from_device
```

Comment:
Probably should call this 'garage.global_device' or something, so that it definitely doesn't conflict with any sub-field names.